index.js

```diff
@@ -18,19 +18,19 @@ const isPortFree = port => new Promise((resolve) => {
     server.close();
   });
 
-  server.listen(port, '0.0.0.0');
+  server.listen(port, host);
 });
 
-const findFreePort = port => isPortFree(port)
+const findFreePort = (port, host) => isPortFree(port, host)
```

Comment:
It's better to update readme, because function signature was changed.

Reply:
Updated description in readme and added default host value for `findFreePort`.